<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColectionerWindowClass</class>
 <widget class="QWidget" name="ColectionerWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColectionerWindow</string>
  </property>
  <widget class="QListWidget" name="ItemsList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>256</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="OffersList">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>256</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>49</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Offer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Bid</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
